frontend/src/constants/feedback-questions.tsx

```diff
@@ -1,318 +1,326 @@
 import { FeedbackFieldTypes } from 'types/wizard-form.d';
+import { t } from '@transifex/native';
 
-const LOCATIONS_LIST = [
-  { id: 'africa', label: 'Africa' },
-  { id: 'asia', label: 'Asia' },
-  { id: 'caribbean', label: 'Caribbean' },
-  { id: 'central_america', label: 'Central America' },
-  { id: 'eastern_europe', label: 'Eastern Europe' },
-  { id: 'european_union', label: 'European Union' },
-  { id: 'middle_east', label: 'Middle East' },
-  { id: 'north_america', label: 'North America' },
-  { id: 'oceania', label: 'Oceania' },
-  { id: 'south_america', label: 'South America' },
-  { id: 'uk', label: 'UK' },
+const getLocationsList = () => [
+  { id: 'africa', label: t('Africa') },
+  { id: 'asia', label: t('Asia') },
+  { id: 'caribbean', label: t('Caribbean') },
+  { id: 'central_america', label: t('Central America') },
+  { id: 'eastern_europe', label: t('Eastern Europe') },
+  { id: 'european_union', label: t('European Union') },
+  { id: 'middle_east', label: t('Middle East') },
+  { id: 'north_america', label: t('North America') },
+  { id: 'oceania', label: t('Oceania') },
+  { id: 'south_america', label: t('South America') },
+  { id: 'uk', label: t('UK') },
 ];
 
-export const INTRO = {
-  title: 'Intro',
-  nextButton: 'Tool Use',
+export const getIntro = () => ({
+  title: t('Intro'),
+  nextButton: t('Tool Use'),
   questions: [
     {
       id: 'work_sector',
       type: FeedbackFieldTypes.Single,
       required: true,
-      question: 'What sector do you work in?',
+      question: t('What sector do you work in?'),
       answers: [
-        { id: 'government', label: 'Government' },
-        { id: 'academia', label: 'Academia' },
-        { id: 'non_profit', label: 'Non-profit' },
-        { id: 'private', label: 'Private' },
-        { id: 'donor', label: 'Donor' },
-        { id: 'media', label: 'Media' },
+        { id: 'government', label: t('Government') },
```

Comment:
LGTM! 

The only caveat I see, is that when the form data is sent to the API, the API expects the questions to be in English; not the translated versions. Would there be a way to access the original string in English, when processing the form data, without duplicating these labels? 

eg: `{ id: 'government', label: t('Government') }, label_en: 'Government'`

Reply:
I don't think so if we get them from the current form status. I think we will have to duplicate them